repo-checklist.md

```diff
@@ -27,5 +27,6 @@ After copying the contents of this repo into the new repo-to-be, follow the belo
    1. Make all members of the team maintainers (so they can add new maintainers as needed).
    1. Give the team write access to the new repository.
 1. Create a maintainers' mailing list. There are no requirements on mailing list providers, but it needs to be invite only, readable by members only, but accept mails from non-members. [Google Groups](http://groups.google.com/) can easily set up such mailing lists.
+   1. Add all [TC members](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members) as owners in this mail-list.
```

Comment:
Nit pick: We used the word "mailing list" in the paragraph before but now it says "mail-list" in this sentence.

Reply:
```suggestion
   1. Add all [TC members](https://github.com/eiffel-community/community/blob/master/GOVERNANCE.md#technical-committee-members) as owners in this mailing list.
```